include/cupla/manager/Stream.hpp

```diff
@@ -165,7 +147,27 @@ namespace manager
         {
         }
 
-        //! unique if for the next stream
+        auto
+        createNewStream( cuplaStream_t streamId  )
+        -> cuplaStream_t
+        {
+
+            auto& device = Device< DeviceType >::get();
+
+            std::unique_ptr<
+                QueueType
+            > streamPtr(
+                new QueueType(
+                    device.current()
+                )
```

Comment:
`std::make_unique` is in C++14.

Reply:
I update it, it was code from c++11 times.